tools/apidiff/exports/exports.go

```diff
@@ -110,10 +118,15 @@ func (c *Content) addConst(pkg Package, g *ast.GenDecl) {
 				// const FooConst = FooType("value")
 				co.Type = pkg.getText(ce.Fun.Pos(), ce.Fun.End())
 				v = pkg.getText(ce.Args[0].Pos(), ce.Args[0].End())
+			} else if ce, ok := vs.Values[0].(*ast.BinaryExpr); ok {
+				// const FooConst = "value" + Bar
+				co.Type = "*ast.BinaryExpr"
```

Comment:
Why do we need this?

Reply:
If I recall correctly, the Type field can't be empty so I put this arbitrary value in it. I could put the content of the const in it instead if you think that's better?

Reply:
Oh yes I remember now.  It's used during comparison to see if the `const` type has changed.  I think what you have is fine.